docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,378 @@
+# Installing With SR-IOV Worker Nodes
+
+## Table of Contents
+
+- [Pre-Install Requirements](#pre-install-requirements)
+- [Preparing your OpenShift Cluster to use SR-IOV Networks](#preparing-your-openshift-cluster-to-use-sr-iov-networks)
+- [Creating SR-IOV Networks for Worker Nodes](#creating-sr-iov-networks-for-worker-nodes)
+- [Creating SR-IOV Worker Nodes in IPI](#creating-sr-iov-worker-nodes-in-ipi)
+- [Creating SR-IOV Worker Nodes in UPI](#creating-sr-iov-worker-nodes-in-upi)
+
+## Pre-Install Requirements
+
+Using single root I/O virtualization (SR-IOV) networking as an additional network in OpenShift can be beneficial for applications
+that require high bandwidth and low latency. If you are planning an OpenStack deployment with SR-IOV NICs enabled, refer to [the OSP 16.1 installation documentation](osp-sriov-install). Before installing, make sure that the network interface cards your OpenStack nodes are using [are supported](supported-nics) for use with SR-IOV
+in OpenShift, and that your tenant has access to them. Your OpenStack cluster will also need the following requirements for each SR-IOV instance:
+
+- An instance from the RHOSP quota
+- Two ports from the RHOSP quota
+- A flavor with at least 16 GB memory, 4 vCPUs, and 25 GB storage space
+
+Additionally, for clusters that use single-root input/output virtualization (SR-IOV), RHOSP compute nodes require a flavor that supports [huge pages](huge-pages).
+Deploying worker nodes with SR-IOV networks is supported as a post install operation in the installer for both IPI and UPI workflows. Once you have verified that your OpenStack cluster can support SR-IOV in OpenShift, and have succesfully installed an OpenShift cluster that meets the [minimum requirements](README.md#openstack-requirements), you can follow these steps and example workflows to create worker nodes with SR-IOV NICs.
+
+## Preparing your OpenShift Cluster to use SR-IOV Networks
+
+Before you use single root I/O virtualization (SR-IOV) on a cluster that runs on Red Hat OpenStack Platform (RHOSP), make the RHOSP metadata service mountable as a drive and enable the No-IOMMU Operator for the virtual function I/O (VFIO) driver.
+
+### Enabling the RHOSP metadata service as a mountable drive
+
+Next, you will need to create a machine config that enables the SR-IOV operator to get the UUID of networks from OpenStack. This machine config will allow the OpenStack metadata service to be available as a mountable drive.
```

Comment:
"worker machineconfig" I think this changes have to be made to the machine config describing the workers right?  its obvious, but best to spell it out.

Reply:
no, its actually its own machine config that is attached to the worker machinepool definition